packer/vhd-image-builder.json

```diff
@@ -51,12 +51,12 @@
         },
         {
             "type": "file",
-            "source": "parts/k8s/kubernetesinstalls.sh",
+            "source": "parts/k8s/cloud-init/artifacts/cse_install.sh",
```

Comment:
proposed renaming to more clearly express what this file contains

Reply:
The only thing I don't like about using the word "cse" in there is because this file is called by both the CSE flow and the Packer VHD builder flow.

Reply:
I like putting the sh files in /cloud-init/artifacts though. I guess "cse" could make sense if we look at it from the perspective of "VHD is just pre-running some CSE install scripts". Playing the devil's advocate here...

Reply:
My thinking is that the shell scripts are designed for execution by CSE, so the fact that packer invokes them is a re-use win for us. But I think the foundational thing we're doing here is the "implement for CSE" thing.

Reply:
I can get behind that. What does @mboersma think?

Reply:
"VHD is just pre-running some CSE install scripts" is in the neighborhood of logical in my opinion. :+1: